lib/agoneum/account/account.ex

```diff
@@ -79,9 +98,9 @@ defmodule Agoneum.Account do
       {:error, %Ecto.Changeset{}}
 
   """
-  def update_user(%User{} = user, attrs) do
+  def update_user(%User{} = user, attrs, type \\ :html) do
```

Comment:
I need to add a `when is_atom(type)` guard to this, assuming the default value gets set before the guard check